### DISCUSSION TITLE
Radeon ProRender for Maya 3.5.1

### DISCUSSION THREAD

**Discussion by bsavery** [GPUOpen-LibrariesAndSDKs/RadeonProRenderMayaPlugin#385] [Announcements]
Radeon ProRender 3.5.1 for Maya, enhances both viewport and final rendering modes:

- RPR Final uses HIP instead of OpenCL.
- RPR Interactive (using Vulkan and hardware ray tracing) now includes RESTIR-based lighting calculations and FSR upsampling for faster interactive viewport rendering.

Plus many more changes.  See the [changelog](https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderMayaPlugin/blob/master/CHANGELOG.md) for more details.

<hr /><em>This discussion was created from the release <a href='https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderMayaPlugin/releases/tag/v3.5.0'>Radeon ProRender for Maya 3.5.1</a>.</em>

**Comment by imansham** [comment]
Pro Render doesn't work in Maya 2024 or higher. It isn't even listed in the plugins to enable. Please do something..................

**Comment by hbecerra81** [comment]
I can confirm that it is not working in 2024 either. I even tried to manually add it from the 2023 directory, but it won't load the ".mll". Says the module is not found. Happy to help troubleshoot this to get it working. I have dual AMD Radeon Pro WX 8200 GPUs.

**Reply by hbecerra81** [reply]
@bsavery  Hey, can you please enlighten us on this issue? Are we doing something wrong, or is this just not going to get updated for a solution ever?
